content/zh-cn/blog/_posts/2017-10-00-Five-Days-Of-Kubernetes-18.md

```diff
@@ -3,14 +3,11 @@ title: " Kubernetes 1.8 的五天 "
 date: 2017-10-24
 slug: five-days-of-kubernetes-18
 ---
-
 <!--
----
 title: " Five Days of Kubernetes 1.8 "
 date: 2017-10-24
 slug: five-days-of-kubernetes-18
-url: /zh/blog/2017/10/Five-Days-Of-Kubernetes-18
----
+url: /blog/2017/10/Five-Days-Of-Kubernetes-18
```

Comment:
It can be added to metadata too.

Reply:
yes, it can be added, but can choose on of slug or url. If both exists, url will override slug, and keep case.